led_strip/api.md

```diff
@@ -131,8 +131,8 @@ esp_err_t led_strip_set_pixel_hsv (
 - `strip` LED strip
 - `index` index of pixel to set
 - `hue` hue part of color (0 - 360)
-- `saturation` saturation part of color (0 - 255)
-- `value` value part of color (0 - 255)
+- `saturation` saturation part of color (0 - 255, rescaled from 0 - 1)
+- `value` value part of color (0 - 255, rescaled from 0 - 1)
```

Comment:
this file actually was auto-generated from the header file.. Use the way suggested in https://github.com/espressif/idf-extra-components/issues/248

Reply:
so let's update the header file first, and then sync the changes to api.md

Reply:
can we give the user an example in the API doc, saying if the situation is 50%, how should he set the value here